README.md

```diff
@@ -134,11 +134,13 @@ int main() {
 To create the parser, compile this file like this:
 
 ```sh
+libdir=$(dirname $(find tree-sitter/out/Release -name libcompiler.a))
+
 clang++ -std=c++11 \
   -I tree-sitter/include \
-  -L tree-sitter/out/Release \
-  -l compiler \
+  -L "$libdir" \
```

Comment:
Maybe the simplest approach is to pass the `.a` file as part of the object-file list rather than using the linker flags `-l` and `-L` at all.

```sh
clang++ -std=c++11 \
  $(find tree-sitter/out -name libcompiler.a)  \
  arithmetic_grammar.cc \
  -o arithmetic_grammar
```

This works for me on macOS using clang? Does this work on your linux system?

Reply:
WFM! Given the fix to the order too, that is.

I think it'd be slightly cleaner also to refer to `tree-sitter/out/Release` rather than `tree-sitter/out` -- that way if the person has also e.g. run tests, it still finds just one file. On my system it seems to work with no complaint even if two versions of libcompiler.a show up on that command line, but I'd worry a bit about counting on that in general.

I'll send a version with this tweak.

Reply:
Sent. Also fixed where I was missing `""` around the `find` results.